pages/team-structure.js

```diff
@@ -0,0 +1,402 @@
+
+    constructor(props){
+        super(props)
+        // NOTE: selectedMemberId -- each tier's card has a unique id
+        // selectedTier -- each tier has a unique ID
+        this.state = {
+            memberSelected: false,
+            selectedMemberId: undefined,
+            selectedTierId: 1,
+            selectedTeam: undefined,
+        };
+    }
+
+    onSelectMember = (cardId, tierId) => {
+        // TODO: put a condition ??
+        if (tierId === 2) {
+            const selectedSubTeamCard = subteam_json.filter(function(subTeamCard){return subTeamCard._id === cardId})[0]
+            this.setState({
+                selectedTierId: tierId, 
+                selectedMemberId: selectedSubTeamCard._id, 
+                memberSelected: true, 
+                selectedTeam: selectedSubTeamCard._id
+            })
+        } else {
+            this.setState({
+                selectedTierId: tierId, 
+                selectedMemberId: cardId, 
+                memberSelected: true, 
+                selectedTeam: undefined
+            })
+        }
+    }
+
+    render() {
+        return (
+            <PageTemplate title="Team Structure">
+                <React.Fragment>
+                <ThemeProvider theme={teamHierCustomTheme}>
+                    <SystemComponent 
+                        display="flex"
+                        overflow="hidden"
+                        height="100%"
+                        border="3px solid black"
+                    >
+                        <SystemComponent 
+                            width="240px"
+                            height="100%"
+                            bg="#D6D6D6"
+                        >
+                            <TeamStructSideNav></TeamStructSideNav>
+                        </SystemComponent>
+                        
+                        <TeamHierParentContainer pl={28} pr={28}>
+                            <TeamHierFormWrapper></TeamHierFormWrapper>
+                                
+                            <SystemComponent display="flex" flexDirection="column" overflowX="hidden">
+                                
+                                <HierSubSection>
+                                    <Header3>Directors</Header3>
+                                    <TeamHierMemberListGrid tierId={1} roleCards={director_json} onSelect={this.onSelectMember}></TeamHierMemberListGrid>
+                                </HierSubSection>
+
+                                <HierSubSection>
+                                    <Header3>Team Leads</Header3>
+                                    <TeamHierMemberListGrid tierId={2} roleCards={subteam_json} onSelect={this.onSelectMember}></TeamHierMemberListGrid>
+                                </HierSubSection>
+
+                                <HierSubSection>
+                                    <Header3>Project Heads</Header3>
+                                    <GridListParentContainer 
```

Comment:
Instead of hiding project leads and members until you click on someone, could we show every single member and when clicking on a lead, it will filter the members

Reply:
Will add a TODO line. This will require some changes to the logic, but I don't want to rush to get this in since today (Nov 30) is the deadline for the first version's completion.